terraform/environments/bootstrap/member-bootstrap/iam.tf

```diff
@@ -40,6 +40,10 @@ data "aws_iam_policy_document" "member-access" {
       "cloudtrail:AddTags",
       "cloudtrail:CancelQuery",
       "cloudtrail:Create*",
+      "cloudtrail:StartLogging",
+      "cloudtrail:StopLogging",
```

Comment:
```suggestion
```
Could we remove the StopLogging action please?  I'm wary of giving permission to turn off Cloudtrail logs! Thanks

Reply:
removed 

      "cloudtrail:StopLogging",